Sources/XMTP/Messages/Invitation.swift

```diff
@@ -1,35 +1,44 @@
 //
 //  Invitation.swift
 //
-//
-//  Created by Pat Nakajima on 11/26/22.
-//
 
 import Foundation
 
 /// Handles topic generation for conversations.
 public typealias InvitationV1 = Xmtp_MessageContents_InvitationV1
 
 extension InvitationV1 {
-	static func createRandom(context: InvitationV1.Context? = nil) throws -> InvitationV1 {
```

Comment:
@nakajima and I talked about there still being a need for `createRandom` And so `createDeterministic` wouldn't necessarily remove it but we can always add it back.